Weapon Mods/Stable/Starcore-MA-Weapons-Natemod/Data/Scripts/CoreParts/MA_Laser_T2.cs

```diff
@@ -226,7 +226,7 @@ partial class Parts {
                         MuzzlePartId = "T2_EL", // The subpart where your muzzle empties are located.
                         AzimuthPartId = "T2_AZ1",
                         ElevationPartId = "T2_EL",
-                        DurabilityMod = 0.25f, // GeneralDamageMultiplier, 0.25f = 400% resistance.
+                        DurabilityMod = 0.2f, // GeneralDamageMultiplier, 0.25f = 400% resistance.

```

Comment:
is this intentional?

Reply:
Yeah if the gimble is going to be the same price as the pdx and the pdx has a 400% resist with not a lot of hp, then the gimble can have a 400% resist too with its not a lot of hp